CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Add `compatibility_issues` method to `ConfluentSchemaRegistry` to debug compatibility issues between a schema versions for a given subject (#212)
```

Comment:
I popped it into this section, but let me know if you wanted me to add to a new `v1.18.0` heading instead 😁

Reply:
This is perfect!